cmd/gaia/init/genesis_accts.go

```diff
@@ -28,7 +29,15 @@ func AddGenesisAccountCmd(ctx *server.Context, cdc *codec.Codec) *cobra.Command
 
 			addr, err := sdk.AccAddressFromBech32(args[0])
 			if err != nil {
-				return err
+				kb, err := keys.GetKeyBaseFromDir(viper.GetString(flagClientHome))
```

Comment:
OK, this is convenient - I wonder if we should *just* support named aliases, but that can be a later question...

Reply:
Yes, I'm doing it just for convenience. But I wouldn't frown upon a change request and I reckon this could we be seen as superfluous.